apps/api/src/graphql/schema/programs.ts

```diff
@@ -66,10 +66,18 @@ input ProgramRequirementsQuery {
     programId: String!
 }
 
+input MajorIdQuery {
+    majorId: String!
```

Comment:
If you split this type as I suggest below, then the field for the majors query should be named just `id`.